### PR TITLE
Move list style to UL and removed padding

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do? (please provide any background)
- `list-style` should not be added to an `li` it should be added to the `ul` is it conatiner within
- I have also removed default padding from the icon list
- This is because on the docs (with bootstrap) everything looked okay, but in reality it wasn't quite right

#### What tests does this PR have?
None

#### How can this be tested?
- Clone branch
- run `npm run docs` and check that the lost icon looks aligned and new styles are present (see screenshot)

#### Screenshots / Screencast
screen shot 2015-11-23 at 14 22 25

#### What gif best describes how you feel about this work?
Head BANG

---

- [x] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.

#### Reviewers

**Review 1**
- [x] :+1:

**Review 2** \*
- [x] :+1:

**Review 3** _(optional)_
- [ ] :+1:

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru

### CODE REVIEW

File: docs/public/assets/bundle.js [hunk 1]
```diff
@@ -27765,7 +27765,7 @@ module.exports = require('./lib/React');
 },{"./lib/React":146}],279:[function(require,module,exports){
 module.exports={
   "name": "ui-toolkit",
-  "version": "0.25.0",
```
Comment (jodiedoubleday) [inline docs/public/assets/bundle.js]: I'm not sure why this has updated, this should stay at `0.25.0` unless someone hasn;t re-built on master properly

Comment (robhuzzey) [inline docs/public/assets/bundle.js]: my bad I think :(

**Comment by robhuzzey** [comment]
Looks good to me... I've released master patch `0.26.1` for the build issue too. :+1:

**Comment by jackdcrawford** [comment]
:+1:

**Pull request merged**